apps/desktop/src/lib/commit/StackingCommitList.svelte

```diff
@@ -108,7 +108,7 @@
 		isMiddle?: boolean;
 		isLast?: boolean;
 	}) {
-		if (isFirst) return 12;
+		if (isFirst) return 0;
```

Comment:
Does this behavior need to be dependent on the stacking flag?

Reply:
No, it's in the `StackingCommitList` specifically which is only used here. But actually now that you pointed it out, that entire calculating of `yOffset` can be simplified here